tools/pubsys-config/src/lib.rs

```diff
@@ -111,6 +111,10 @@ impl InfraConfig {
 #[derive(Debug, Default, Deserialize, Serialize, PartialEq, Eq, Clone)]
 pub struct Vendor {
     pub registry: String,
+    // Needed in cases where the vendor does not use HTTPS. This is to accomodate a weird quirk in
+    // the implementation of `docker manifest create`.
+    #[serde(default)]
+    pub insecure: bool,
```

Comment:
Let's get rid of this - for now we just won't support registries that don't use HTTPS.